problem_builder/public/js/mentoring.js

```diff
@@ -8,7 +8,9 @@ function MentoringBlock(runtime, element) {
     var attemptsTemplate = _.template($('#xblock-attempts-template').html());
     var data = $('.mentoring', element).data();
     var children = runtime.children(element);
-    var steps = runtime.children(element).filter(function(c) { return c.element.className.indexOf('assessment_step_view') > -1; });
+    var steps = runtime.children(element).filter(function(c) {
+        return $(c.element).attr("class").indexOf('assessment_step_view') > -1;
+    });
```

Comment:
In case you're wondering why you had to make this change: https://openedx.atlassian.net/browse/TNL-4161

I believe the earlier version is "better" but it doesn't work with the latest edx-platform master. The new version you have here should always work, so we may as well keep this change.